tests/test_batch_change.py

```diff
@@ -191,6 +191,44 @@ def test_reject_batch_change(mocked_responses, vinyldns_client):
     check_batch_changes_are_same(r, bc)
 
 
+def test_cancel_batch_change(mocked_responses, vinyldns_client):
+    error_message = (
+        "Zone Discovery Failed: zone for 'foo.bar.com' does not exist in"
+        " VinylDNS. If zone exists, then it must be connected to in VinylDNS."
+    )
+
+    error = ValidationError('ZoneDiscoveryError', error_message)
+
+    arc = AddRecordChange(
+        forward_zone.id, forward_zone.name, 'cancel', 'cancel.bar.com',
+        RecordType.A, 200, AData('1.2.3.4'), 'PendingReview', 'id1',
+        [error], 'system-message', 'cchangeid1', 'csid1'
+    )
+
+    drc = DeleteRecordSetChange(
+        forward_zone.id, forward_zone.name, 'cancel2', 'cancel2.bar.com',
+        RecordType.A, 'Complete', 'id2', [], 'system-message', 'cchangeid2',
+        'csid2'
+    )
+
+    bc = BatchChange(
+        'user-id', 'user-name', datetime.utcnow(), [arc, drc], 'bcid',
+        'Cancelled', 'Cancelled', comments='cancel_batch_change test',
+        owner_group_id='owner-group-id', reviewer_id='admin-id',
+        reviewer_user_name='admin', review_comment='not good',
+        review_timestamp=datetime.utcnow()
```

Comment:
remove `comments`,  `reviewer_id`, `reviewer_user_name`, `review_comment`, and `review_timestamp` from here.